packages/resourceful-redux/test/unit/reducers/delete.js

```diff
@@ -317,6 +361,7 @@ describe('reducers: delete', function() {
           3: null,
           4: null
         },
+        lists: {},
```

Comment:
would it be better for the tests to do something like `Object.assign(defaultSlice, whatTestIsDoing)` so that we don't have to add new top level keys to _every_ test when we do this?

Reply:
Yes, great idea!

Reply:
Gonna do this in a follow-up PR